extensions/3DTILES_implicit_tiling/0.0.0/schema/3DTILES_implicit_tiling.content.schema.json

```diff
@@ -2,15 +2,15 @@
   "$schema": "http://json-schema.org/draft-04/schema",
   "title": "Content",
   "type": "object",
-  "description": "An object describing the type, availability, and location of tile content.",
+  "description": "An object describing the type and location of tile content.",
   "properties": {
     "mimeType": {
       "type": "string",
       "description": "The mime type of the content."
     },
     "uri": {
       "allOf": [{"$ref": "templateUri.schema.json"}],
-      "description": "A template URI pointing to tile content. Content exists wherever the availability bit is 1. The URI of each content is substituted with the tile's global level, x, and y. For tiling scheme `OCTREE`, z must also be given. All content must conform to the `mimeType`."
+      "description": "A template URI pointing to tile content. Content exists wherever the availability bit is 1. The URI of each content is substituted with the tile's global level, x, and y. For tiling scheme `OCTREE`, z may also be given. All content must conform to the `mimeType`."
```

Comment:
Changed `must` to `may` for `${z}`. I think it's ok if the tileset author has flexibility in creating the template string, including leaving out the templates entirely.